CompilerSource/compiler/compile.cpp

```diff
@@ -112,8 +112,42 @@ inline void write_desktop_entry(const std::string fPath, const GameSettings& gam
   wto << "Categories=Game;\n";
   wto << "MimeType=application/octet-stream;\n";
   wto.close();
-  
+
+  #if defined(__linux__)
```

Comment:
i told you ifndef win32 idiot. windows is the only os without chmod. osx has it

Reply:
Does it have the same concept of executable files though?

Reply:
yes, in all fillesystems other than ntfs ad fat32 theres an extecutable bit on files. Windows just dumb

Reply:
Is it at all possible on Linux to download a file already chmod'd to be executable? What exactly does chmod do? Does it change anything in the actual bytes that comprise the file or is it changing something in the operating system's files/the file system's table?

Reply:
Asked Josh, and I have changed it: 2595c1da18a2843b80faa595263dd8eb562f4d5b